bot/main.ts

```diff
@@ -0,0 +1,86 @@
+#!/usr/bin/env -S node --no-warnings --loader ts-node/esm
+/**
+ * Wechaty - Conversational RPA SDK for Chatbot Makers.
+ *  - https://github.com/wechaty/wechaty
+ */
+import {
+    Contact,
+    Message,
+    ScanStatus,
+    Wechaty,
+    log,
+} from 'wechaty'
+
+import qrcodeTerminal from 'qrcode-terminal'
+
+// https://stackoverflow.com/a/42817956/1123955
+// https://github.com/motdotla/dotenv/issues/89#issuecomment-587753552
+import 'dotenv/config.js'
+
+function onLogout(user: Contact) {
+    log.info('StarterBot', '%s logout', user)
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/philip-zhan/badminton-group/pullRequest?prid=8160404)
```suggestion
    log.info("StarterBot", '%s logout', user)
```